lib/spack/spack/util/compression.py

```diff
@@ -22,6 +22,22 @@ def allowed_archive(path):
     return any(path.endswith(t) for t in ALLOWED_ARCHIVE_TYPES)
 
 
+def _gunzip(archive_file):
+    """Like gunzip, but extracts in the current working directory
+    instead of in-place.
+
+    Args:
+        archive_file (str): absolute path of the file to be decompressed
+    """
+    import gzip
```

Comment:
Do all python installations support this?

Reply:
@becker33 Not sure how to verify this. This module is part of the standard library, and it's present in the Python 2.6 interpreters I have access to. That said I don't know if there are distributions that package it separately like e.g. Ubuntu does for `python-tk`.

An alternative would be to use the old command but try to move the extracted file into place.